lib/warnings.js

```diff
@@ -6,6 +6,8 @@ const { createWarning } = require('process-warning')
  * Deprecation codes:
  *   - FSTWRN001
  *   - FSTSEC001
+ *
+ * Next deprecation code: FSTDEP022.
```

Comment:
```suggestion
 * Deprecation Codes FSTDEP001 - FSTDEP022 were used by v4 and MUST NOT not be reused.
```

Using must not in capital letters in regards of RFC 2119